platforms/extensibility/unit-test-fixtures/build.gradle.kts

```diff
@@ -47,6 +47,8 @@ dependencies {
     implementation(projects.native)
     implementation(projects.serviceRegistryBuilder)
 
+
+    testImplementation(testFixtures(projects.core))
```

Comment:
We need this dependency to be able to use `TestUtil.problemService()` in `ProjectBuilderTest`.